src/components/HeaderProfile/HeaderProfile.test.jsx

```diff
@@ -0,0 +1,38 @@
+import { render, screen } from "@testing-library/react";
+import { customRender } from "../../utils/testUtils";
+import HeaderProfile from "./HeaderProfile";
+import {ReactComponent as Profile} from "../../assets/profile-picture.svg"
+
+it("Should render the HeaderProfile", () => {
+  const { container } = customRender(
+    <HeaderProfile profileImg={Profile}/>
+  );
+  expect(container).toMatchSnapshot();
+});
+
+it("Should render the svg in the Header Profile", () => {
+  render(
+    <HeaderProfile profileImg={Profile}/>
+  );
+  const svg = screen.getByAltText("profile picture");
+    !expect(svg).toBeInTheDocument();
```

Comment:
const svgAltText = screen.queryByText("user avatar");  <- you want to just check if the text is displayed somewhere, not an alt text, as this one is looking for it in HTML, so it's always there
expect(svgAltText).toBeFalsy();